sdv/metadata/single_table.py

```diff
@@ -55,6 +55,8 @@ class SingleTableMetadata:
     }
 
     _NUMERICAL_REPRESENTATIONS = frozenset([
+        'Float32',
+        'Float64',
```

Comment:
I didn't add a test for this because there is already this test to check the valid `computer_representation`:
https://github.com/sdv-dev/SDV/blob/a0e0a76c23c644145b223c7b5ede4a21b31238a0/tests/unit/metadata/test_single_table.py#L120